devenv/docker-compose.yml

```diff
@@ -123,7 +123,7 @@ services:
         GIT_URI: https://github.com/hirosystems/explorer.git
         GIT_BRANCH: v1.119.0
     ports:
-      - 3000:3000
+      - 3020:3020
```

Comment:
```suggestion
      - 3020:3000
```

Reply:
Huh, why would that work?

Reply:
The explorer starts and listenes on port 3000. Then docker maps this to 3020. From the outside it looks like the explorer listens to 3020. From the inside nothing needs to be changed.